vision/google/cloud/vision_v1p2beta1/proto/product_search_service.proto

```diff
@@ -11,39 +11,41 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//
 
 syntax = "proto3";
 
-package google.cloud.vision.v1p3beta1;
+package google.cloud.vision.v1;
```

Comment:
this one is odd. The folder is v1p2beta1, the old protos was v1p3beta1, new one is v1.